bin/gpu-trace.py

```diff
@@ -679,10 +679,11 @@ def StandaloneMain(args):
     State().traceExitEvent.wait()
 
     gpuTrace.CaptureTrace(args.output_dat)
+    perfOk = perfTrace is not None and perfTrace.CaptureTrace(args.perf_json)
```

Comment:
Can we have this as
```
perfStarted = False
if perfTrace is not None:
    perfStarted = perfTrace.CaptureTrace(args.perf_json)
```

I know it is a lot more verbose, but I find that reading the one line solution hides the side-effects of the statement.

Reply:
No problem!  I just pushed an update that's a bit more explicit.